packages/@okta/vuepress-site/docs/reference/api/oidc/index.md

```diff
@@ -1199,14 +1198,16 @@ If more than 100 groups match the filter, then the request fails. Expect that th
-| `id_token`                  | Claims associated with requested scopes.                                                             | N/A                                         |
-| `id_token` `code `          | Claims associated with requested scopes.                                                             | N/A                                         |
-| `id_token` `token`          | `email` if email scope is requested; `name` and `preferred_username` if profile scope is requested   | Claims associated with the requested scopes |
-| `code` `id_token` `token`   | `email` if email scope is requested; `name` and `preferred_username` if profile scope is requested   | Claims associated with the requested scopes |
+  | Response Type             | Claims Returned in ID Token from Default AS                                                        | Claims Returned in ID Token from Custom AS | Claims Returned from Userinfo Endpoint     |
+  | :-------------------------| :------------------------------------------------------------------------------------------------- | :----------------------------------------- | :----------------------------------------- |
+  | `code `                   | N/A                                                                                                | N/A                                        | N/A                                        |
+  | `token`                   | N/A                                                                                                | N/A                                        | N/A                                        |
+  | `id_token`                | Claims associated with requested scopes.                                                           | N/A                                        | N/A                                        |
+  | `id_token` `code `        | Claims associated with requested scopes.                                                           | N/A                                        | N/A
+  | `id_token` `token`        | `email` if email scope is requested; `name` and `preferred_username` if profile scope is requested | Claims associated with the requested scopes and the `alwaysIncludeinToken` property set to `true`. | Claims associated with the requested scopes|
+  | `code` `id_token` `token` | `email` if email scope is requested; `name` and `preferred_username` if profile scope is requested | Claims associated with the requested scopes and the `alwaysIncludeinToken` property set to `true`. | Claims associated with the requested scopes|
+
+> Note: When using a Custom Authorization Server, whether a claim is returned depends on the value set for the `alwaysIncludeinToken` property in the claim configuration. See [Details for `alwaysIncludeinToken`](/docs/reference/api/authorization-servers/#details-for-alwaysincludeintoken) for more information.
```

Comment:
Since we already add a column in the table, this note might not need, we can just add the link for `alwaysIncludeinToken` in the description of the table.